openfasoc/common/platforms/sky130hvl/config.mk

```diff
@@ -97,8 +117,8 @@ export IO_PLACER_H = met3
 export IO_PLACER_V = met2
 
 # keep with gf
-export CELL_PAD_IN_SITES_GLOBAL_PLACEMENT = 4
-export CELL_PAD_IN_SITES_DETAIL_PLACEMENT = 2
+export CELL_PAD_IN_SITES_GLOBAL_PLACEMENT = 1
+export CELL_PAD_IN_SITES_DETAIL_PLACEMENT = 0
```

Comment:
same. Anything customization should be within a generator's config files.

Reply:
Okay got it.